src/helpers/useTodosFilter.ts

```diff
@@ -0,0 +1,22 @@
+import { FilterOption } from '../enum/FilterOption';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  filterSelected: FilterOption,
+};
+
+export const useTodosFilter = ({ todos, filterSelected }: Props): Todo[] => (
```

Comment:
In this function, you make a `filter`, why do you call it `use`
```suggestion
export const filterTodos = ({ todos, filterSelected }: Props): Todo[] => (
```

Reply:
Cause that warn from linter:
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/67290782/c086af70-5317-4425-9870-6478caad30b6)

Reply:
i reimplement that.